model_processing/crates/webify_models/src/image_processing/move_to_textures_dir.rs

```diff
@@ -0,0 +1,80 @@
+//! Move any stray textures to the textures path (typically materials/textures)
+
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+
+use crate::image_processing::Image;
+
+/// Move any stray textures to the textures path (typically materials/textures)
+pub fn move_to_textures_dir<'a>(
+    mut image: Image,
+    base_path: &Path,
+) -> std::result::Result<Image, std::io::Error> {
+    let file_name = image.path.file_name().unwrap().to_str().unwrap();
```

Comment:
I would destructure borrows to the fields of `Image` here.

Reply:
To avoid the double unwraps here you can `and_then`, then handle the none gracefully by turning the option into an error. Something like:

![image](https://user-images.githubusercontent.com/5523172/108642105-244e7e00-7458-11eb-9532-55a818e6e6a8.png)

Reply:
This is something you could do other places in the PR. Maybe you are already aware and which save it future refactor, then keep on rocking.